osmaxx/excerptexport/views.py

```diff
@@ -69,31 +73,91 @@ def get_object(self, queryset=None):
         return o
 
 
-class ExportsListView(LoginRequiredMixin, FrontendAccessRequiredMixin, ListView):
+class ExportsListMixin:
+    @property
+    def excerpt_ids(self):
+        if not hasattr(self, '_excerpt_ids'):
+            self._excerpt_ids = list(
+                OrderedSet(
+                    self.get_user_exports()
+                    .values_list('extraction_order__excerpt', flat=True)
+                )
+            )
+        return self._excerpt_ids
+
+    @property
+    def status_choices(self):
+        allowed_choices = [
+            statuses.FINISHED,
+            statuses.FAILED,
+        ]
+        return [choice for choice in Export.STATUS_CHOICES if choice[0] in allowed_choices]
```

Comment:
The name "allowed choices" here is misleading:

- This list holds just the statuses, **not yet the choices** (those are returned by this method)
- "Allowed" in what sense, exactly? Surely our Exports can have other statuses than just these two.

Should these be `filterable_statuses` or are they also used for something else?